mayastor/src/bdev/nexus/nexus_channel.rs

```diff
@@ -101,7 +101,65 @@ impl NexusChannelInner {
         }
     }
 
-    /// refreshing our channels simply means that we either have a child going
+    /// Remove a child from the readers and/or writers
```

Comment:
Nit: I think we can remove the "or" from here. It looks like we always remove the child from both the readers and writers.

Reply:
Yes, but it is not given that both will have the child, right?